pkg/asset/templates/kubelet-master.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kubelet
+  namespace: kube-system
+  labels:
+    k8s-app: kubelet
+    version: v1.2.4_inotify.3
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: kubelet
+        version: v1.2.4_inotify.3
+    spec:
+      nodeSelector:
+        master: "true"
+      containers:
+      - name: kubelet
+        image: quay.io/peanutbutter/hyperkube:v1.2.4_inotify.3
+        command:
+        - /nsenter
+        - --target=1
+        - --mount
+        - --wd=.
+        - --
+        - ./hyperkube
+        - kubelet
+        - --api-servers={{ index .APIServers 0 }}
+        - --config=/etc/kubernetes/manifests
```

Comment:
Is the only difference with "kubelet-master" this flag? It should be fine to add this to all kubelets. We could either just allow static pods from the hosts at that location (but on nodes we don't really have any right now). Or we could rely on /etc/kubernetes/manifests not actually being mounted on the hosts for workers (then it's just a path inside the container and will only be available during the lifetime of the pod